test/node-api/test_general/test_general.c

```diff
@@ -26,6 +26,57 @@ static napi_value testGetNodeVersion(napi_env env, napi_callback_info info) {
   return result;
 }
 
+static napi_value testBufferFromArrayBuffer(napi_env env,
+                                            napi_callback_info info) {
+  napi_status status;
+  napi_value arraybuffer;
+  void* data;
+  napi_value buffer;
+  size_t byte_length = 1024;
+  size_t byte_offset = 0;
+
+  status = napi_create_arraybuffer(env, byte_length, &data, &arraybuffer);
+  NODE_API_ASSERT(env, status == napi_ok, "Failed to create arraybuffer");
+
+  status = node_api_create_buffer_from_arraybuffer(
```

Comment:
The unit test fails to compile because the new function is an experimental API.
It can be fixed by adding `'defines': ['NAPI_EXPERIMENTAL']` to the test `.gyp` file.
See as an example, https://github.com/nodejs/node/blob/70de19e2ccdd6e39a1b4417d7c614d32d742fdd3/test/node-api/test_threadsafe_function/binding.gyp#L16-L18

Reply:
thanks, ı applied